whylog/assistant/regex_assistant/regex.py

```diff
@@ -0,0 +1,119 @@
+"""
+Regex verification and creating (but not finding groups in regex)
+"""
+import re
+
+special_characters = frozenset(['.', '^', '$', '*', '+', '{', '}', '?', '[', ']', '|', '(', ')'])
+
+
+class NoDateGroupError(Exception):
```

Comment:
It is worth to consider. Propose it on slack, to see what other teammates think about it. it is decision about whole project